src/main/java/com/zendesk/maxwell/row/RowMapDeserializer.java

```diff
@@ -99,12 +105,32 @@ public RowMap deserialize(JsonParser jsonParser, DeserializationContext deserial
 		return rowMap;
 	}
 
-	private Object getValue(ValueNode value)
+	@VisibleForTesting
+	final Object getValue(ValueNode value)
```

Comment:
I wouldn't recommend changing the method accessibility only for testing purpose. In this case it means we can't unit test the method directly but it can still be tested on a higher level.